src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/TdsEnums.cs

```diff
@@ -238,6 +238,7 @@ public enum EnvChangeType : byte
         public const byte FEATUREEXT_DATACLASSIFICATION = 0x09;
         public const byte FEATUREEXT_UTF8SUPPORT = 0x0A;
         public const byte FEATUREEXT_SQLDNSCACHING = 0x0B;
+        public const byte FEATUREEXT_JSONSUPPORT = 0x0D;
```

Comment:
Left a question on the design doc. Curious why this is 0x0D rather than 0x0C when the last extension was 0x0B.

Reply:
@mdaigle  usually the server teams could be working on multiple features, and someone whose feature may come out later may have reserved 0x0C already